earth

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+LAST_CHECK_STATE_PATH=/tmp/last-earth-prerelease-check
+
+function get_latest_binary {
+    docker pull alexcb132/earthlybinaries:latest
+    docker create --name earthly_binary alexcb132/earthlybinaries
+
+    earthbinpath=/earth-linux-amd64
+    if [ "$(uname)" == "Darwin" ]; then
+        earthbinpath=/earth-darwin-amd64
+    fi
+
+    docker cp earthly_binary:$earthbinpath ~/bin/earth-bin
+    docker rm earthly_binary
+}
+
+LAST=`cat $LAST_CHECK_STATE_PATH 2>/dev/null || echo 0`
```

Comment:
```suggestion
LAST=`cat "$LAST_CHECK_STATE_PATH" 2>/dev/null || echo 0`
```

It's a good idea to always put variable expansions in double-quotes. Bash scripts tend to be kinda dangerous.